mltrace/cli/cli.py

```diff
@@ -0,0 +1,19 @@
+import click
+from mltrace.client import (
+    _db_uri,
+    set_address,
+    get_recent_run_ids,
+)
+
+# Set server
+set_address("localhost")
```

Comment:
You may need to make this a command actually, in case people want to change the db address

Reply:
That makes sense!